scripts/extract_glean_client_info_descriptions

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
```

Comment:
This file is missing the `.py` suffix and the license header . Any chance you could fix this before merging?

Reply:
I'm inclined to leave this as-is:

1. The other python scripts don't have .py extensions
2. We don't generally add MPL license headers for internal tooling like this, as we consider the top-level license sufficient context. See e.g. mozilla/gcp-ingestion#777

Reply:
Oh, TIL. Cool, go ahead then!